temba/channels/models.py

```diff
@@ -2007,14 +2007,19 @@ def send_smscentral_message(cls, channel, msg, text):
 
     @classmethod
     def send_vumi_message(cls, channel, msg, text):
-        from temba.msgs.models import WIRED
+        from temba.msgs.models import WIRED, Msg
         from temba.contacts.models import Contact
 
         is_ussd = channel.channel_type in Channel.USSD_CHANNELS
         channel.config['transport_name'] = 'ussd_transport' if is_ussd else 'mtech_ng_smpp_transport'
 
+        if msg.response_to_id:
+            in_reply_to = Msg.objects.get(pk=msg.response_to_id).external_id
```

Comment:
Since this is a tight loop, let's do:

`in_reply_to = Msg.objects.values_list('external_id', flat=True).get(pk=msg.response_to_id)`